tasks/configure.yml

```diff
@@ -41,7 +41,7 @@
 
 - name: PostgreSQL | Reset the cluster - create a new one (with specified encoding and locale) | Debian
   shell: >
-    pg_createcluster --start --locale {{ postgresql_locale }}
+    pg_createcluster --locale {{ postgresql_locale }}
```

Comment:
@nerzhul I agree (with @otakup0pe) this would be a nice change, but must be configurable.
This would be as easy as adding a new default variable like `postgres_cluster_reset_with_start=true` plus replacing ` --start` by 
`{{ postgres_cluster_reset_with_start | ternary('--start','') }}`

`ternary` Ref: See http://docs.ansible.com/ansible/playbooks_filters.html#other-useful-filters